.github/workflows/test.yml

```diff
@@ -20,15 +20,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.15', '1.16' ]
+        go: [ '1.17', '1.18' ]
```

Comment:
Is this reflected in the readme that haaukins only runs on 1.17 and up?

Reply:
it is not reflected, documentation update should come, like
> Working state of this platform is tested on following go versions : [1.17, 1.18] 
> The platform may or may not work for other Go versions. Please take care it at your own risks